src/api/handlers/media_library.py

```diff
@@ -44,12 +44,13 @@ def read_image(self, request):
             return error
         return MassenergizeResponse(data=image_data)
 
-    @admins_only
+    # @admins_only
```

Comment:
is this intentional?

Reply:
I'd also like to know if this is intentional